TINY-10073: copy-edits to remove run-on clauses in API comments/documentation (#8924)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

- Contents would not be removed from the drag start source if dragging and dropping internally into a transparent block element. #TINY-9960
- In some cases pressing enter would scroll the entire page. #TINY-9828
- The border styles of a table were incorrectly split into a longhand form after table dialog updates. #TINY-9843

> - A typo correction plus run-on sentences in API comments/documentation corrected. #TINY-10073

Author: ```suggestion
- API comments/documentation corrected: a typo correction and run-on sentences fixed. #TINY-10073
```
